src/pybind/mgr/dashboard/tests/test_docs.py

```diff
@@ -106,6 +109,12 @@ def test_gen_paths(self):
         # Check that a schema of type 'object' is received in the response.
         self.assertEqual(expected_response_content['202'], outcome['responses']['202']['content'])
 
+    def test_gen_method_paths(self):
+        outcome = Docs()._gen_paths(False)['/api/doctest/']['get']
```

Comment:
Mmmm, but you haven't modified this (fake) protected method (which was the one I suggested)?

Reply:
Modified this now. Thanks